app/views/smart_proxies/plugins/_puppet_envs.html.erb

```diff
@@ -0,0 +1,20 @@
+<% if envs.present? %>
```

Comment:
added empty state here, the partial for environments is not relevant as no envs indicates there is an issue with the puppetmaster setup.